terraform/deployments/datagovuk-infrastructure/remote.tf

```diff
@@ -9,3 +9,28 @@ data "terraform_remote_state" "cluster_infrastructure" {
     region = data.aws_region.current.name
   }
 }
+
+data "terraform_remote_state" "app_govuk_rds" {
+  backend = "s3"
+
+  config = {
+    bucket = var.govuk_aws_state_bucket
+    key    = "blue/app-govuk-rds.tfstate"
+    region = data.aws_region.current.name
+  }
+
+  # TODO: hack because govuk-aws/app-govuk-rds is not terraformed in `test`.
```

Comment:
Feel free to drop the config for the test account if that's easier; it's unlikely we'll ever deploy this stuff there again. (Fine to keep this if you prefer though, whichever's more convenient for now.)

Reply:
ok, I'll drop it